README.md

```diff
@@ -1,15 +1,65 @@
 [![Travis](https://api.travis-ci.org/bids-standard/bids-examples.svg?branch=master "Travis")](https://travis-ci.org/bids-standard/bids-examples)
 
-# BIDS-examples
-A set of BIDS compatible datasets with **empty raw data files** that can be used for writing lightweight software tests.
+# bids-examples
+
+This repository contains a set of BIDS compatible datasets with **empty raw
+data files**. These datasets can be useful to:
+
+1. write lightweight software tests
+1. serve as an example on how a BIDS dataset can be structured
 
 ALL RAW DATA FILES IN THIS REPOSITORY ARE EMPTY!
 
-- You can find full versions of most MRI datasets in this repository [here](https://drive.google.com/drive/u/0/folders/0B2JWN60ZLkgkMGlUY3B4MXZIZW8).
-- You can find the full version of mne_sample_data [here](https://drive.google.com/drive/folders/0B_sb8NJ9KsLUQ3BMS0dxZW5nSHM).
-- You can find the full version of ds000246 (Brainstorm auditory tutorial) [here](https://openneuro.org/datasets/ds000246/versions/00001).
-- You can find the full version of ieeg_epilepsy (Brainstorm epileptogenicity tutorial) [here](https://openneuro.org/datasets/ds001779).
+To learn more about the Brain Imaging Data Structure (BIDS), visit
+http://bids.neuroimaging.io/
+
+If you want to contribute a dataset to the examples, we would be happy to
+accommodate it, as long as it exhibits some structure or particularity that is
+not covered by the already present examples so far.
+
+# Dataset index
 
-To learn more about the Brain Imaging Data Structure (BIDS), visit http://bids.neuroimaging.io/
+If you want to update this table, please open a new issue. For simple editing,
+this table is maintained in a [Google sheet](https://docs.google.com/spreadsheets/d/1d9KfmaKvTOpFWgi0Kd-tZGgVTsZk_4BHH9prcgFeigw/edit?usp=sharing), which is then converted to Markdown for this README file.
 
-If you are looking for datasets with particular features, you can check out the documentation in the [WIKI](https://github.com/bids-standard/bids-examples/wiki/), which is sorted by imaging modalities.
+|  name | description | mri | meg | eeg | ieeg | full data |
+| --- | --- | --- | --- | --- | --- | --- |
+|  7t_trt | field maps, physiological data, quantitative T1 maps, T1w, BOLD | x |  |  |  | https://bit.ly/2H0Z6Qt |
+|  ds000117 | A multi-subject, multi-modal human neuroimaging dataset of 19 subjects on a MEG visual task | x | x | x |  | https://openneuro.org/datasets/ds000117/ |
+|  ds000246 | Auditory dataset used for Brainstorm’s general online tutorial | x | x |  |  | https://openneuro.org/datasets/ds000246/versions/00001 |
+|  ds000247 | Five minutes, eyes-open, resting-state MEG data from 5 subjects. This is a sample from The Open MEG Archive (OMEGA). | x | x |  |  |  |
+|  ds000248 | MNE sample data: Data with visual and auditory stimuli | x | x |  |  | https://bit.ly/2JfNYkf |
+|  ds001 | single task, multiple runs, in-plane T2, events, T1w, BOLD | x |  |  |  | https://bit.ly/2H0Z6Qt |
```

Comment:
I wonder whether the datasets with the 2H0Z6Qt bitly link should also refer to openneuro. I think  that all of these originated from openfmri, but I don't know whether they have been curated/maintained there.

perhaps specify it as  
````
https://bit.ly/2H0Z6Qt or https://openneuro.org/datasets/dsXXXXX
````
for each of them (where XXX is e.g. ds001, although I now see that they have all been zero-padded, so we should link to the zero-padded URL)

Reply:
yes, that's much better - thanks!